_docs/_latest/configure/scanner/rules.md

```diff
@@ -557,7 +559,9 @@ To learn more, see the
 [ForwardingRules](https://cloud.google.com/compute/docs/reference/latest/forwardingRules)
 documentation.
 
-## IAP rules
+## Cloud IAP rules
```

Comment:
Same as before, we'll need to search for any direct links to this heading in our code or docs.

Reply:
As above, possible redirect? Otherwise, yes, we'll need to update for direct links.

Reply:
Resolved.